### PR TITLE
Add desktop, soc, iot, server pages

### PR DESCRIPTION
**Pull request by b-m-f**
# Done
Finishes `/desktop`, `/iot`, `/soc` and `/server` pages

Fixes https://github.com/canonical-web-and-design/base-squad/issues/622

# QA
`./run` and compare the 4 pages from above with the corresponding pages on certification.ubuntu.com

### CODE REVIEW

File: webapp/views.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+from flask import render_template
```
Comment (tbille) [inline webapp/views.py]: the goal of all those views in the future is to move them to ubuntu.com. Could you create a blueprint for them so it's easier to move it from a project to another?

Comment (b-m-f) [reply webapp/views.py]: sure  :)

Comment (b-m-f) [reply webapp/views.py]: @tbille updated it

File: templates/desktop.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    </p>
+    <p class="large">
+      The certification chart below highlights OEM partner PCs available
+      preloaded with Ubuntu. Certified OEM images are also available for Ubuntu
+      Advantage customers.
+    </p>
+
+    <div class="twelve-col cosy-fit">
+      <div class="three-col">
+        <form action="/desktop/models/" method="get" class="body-search">
```
Comment (tbille) [inline templates/desktop.html:45]: can you replace links with `url_for('NAME OF THE FUNCTION')` this also makes it easier when we decide to change some path

Comment (tbille) [reply templates/desktop.html:45]: (same for all href etc.)

File: webapp/views.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    get_releases,
+    get_socs,
+    get_desktops,
+    get_servers,
+    get_iot,
+    get_laptops,
+)
+
+
+def certification_blueprint():
```
Comment (tbille) [inline webapp/views.py]: no need for a factory here.

File: webapp/views.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        static_folder="/static",
+    )
+
+    @certification.route("/desktop")
+    def desktop():
+        release_data = get_releases().json()
+        releases = [
+            release
+            for release in release_data.get("objects")
+            if release.get("desktops") != "0" or release.get("laptops") != "0"
```
Comment (tbille) [inline webapp/views.py]: `if not release.get("desktops") or not release.get("laptops")`

Comment (tbille) [reply webapp/views.py]: oh no it's a string. Forget about it :)

File: templates/desktop.html (hunk 2 above)
Comment (b-m-f) [reply templates/desktop.html:45]: Ah nice! Will update all the ones that already have functions. Ill put it into my TODO to update the other links as soon as their views are ready :)

**Review by tbille — APPROVED** [review]
seems fine to me

**Pull request merged**